src/routes.tsx

```diff
@@ -27,48 +28,51 @@ export const routes: RouteObject[] = [
       ...RouteUtils.getEnabledParaTimes()
         .map((paraTime): RouteObject[] => [
           {
-            path: `/${paraTime}`,
+            path: `/:${paraTime}`,
```

Comment:
This is a bit hacky, could have been just replaced with `:paraTime` and use custom hook, that would consume `useParams()`

Reply:
imo this "hacky way" is an antipattern. Does it even work correctly?
- there is no point in creating dynamic params specific to each paraTime. 
- generated routes look like: `/:emerald`, `/:sapphire` etc. At the end all URLs will match whatever will be the first route in a loop based on `enabledParaTimes` order. For `/sapphire` param route will match Emerald because `:emerald` param will be true and it will be equal to `sapphire`

Reply:
Right, agree that this is an anti-pattern. And as you pointed out, it does not work as expected